st2client/st2client/commands/action.py

```diff
@@ -246,22 +246,6 @@ class ActionRunCommandMixin(object):
     def get_resource(self, ref_or_id, **kwargs):
         return self.get_resource_by_ref_or_id(ref_or_id=ref_or_id, **kwargs)
 
-    @add_auth_token_to_kwargs_from_cli
```

Comment:
Hm, any particular reason why you re-ordered those methods? It's slightly confusing, or did you also make any other changes?